pkg/services/cpd/cpd.go

```diff
@@ -41,36 +42,45 @@ func GetCPDAlertInternalID(alertTitle string) (string, error) {
 // The reasoning for this is that we don't fully trust network verifier yet.
 // In the future, we want to automate service logs based on the network verifier output.
 func InvestigateTriggered(r *investigation.Resources) error {
+	var notesSb strings.Builder
+
+	notesSb.WriteString("CAD CPD VERIFICATION CHECKS\n")
+	notesSb.WriteString("===========================\n")
+
 	if r.Cluster.Status().State() == "ready" {
-		logging.Infof("This cluster is in a ready state and already provisioned")
-		err := r.PdClient.AddNote("This cluster is in a ready state and already provisioned")
+		notesSb.WriteString("⚠️ This cluster is in a ready state, thus provisioning succeeded\n.")
+		err := r.PdClient.AddNote(notesSb.String())
 		if err != nil {
 			logging.Error("could not add clusters ready state to incident notes")
 		}
 	}
+	notesSb.WriteString("✅ Cluster installation did not yet finish\n")
 
 	// Check if DNS is ready, exit out if not
 	if !r.Cluster.Status().DNSReady() {
```

Comment:
**thought:** In this case we should probably let CAD resolve the alert, but I doubt this case will ever trigger.

Reply:
Agreed but I would suggest an iterative approach, once the first SRE encounters it we can review the condition.